setup.py

```diff
@@ -1,7 +1,17 @@
 import os
+import sys
 import re
 from setuptools import setup, find_packages
 
+if sys.platform == 'win32':
+    import ctypes
+
+try:
+    is_root_user = os.geteuid() == 0
+except AttributeError:
+    is_root_user = ctypes.windll.shell32.IsUserAnAdmin() != 0
```

Comment:
I suggest this line to be explicitly checked for work on windows only.

Something like:
```is_root_user = sys.platform == 'win32' and ctypes.windll.shell32.IsUserAnAdmin() != 0```

Reply:
@LD250  Are you just talking about line 12?

Reply:
Yes. Thank you for changes.